stubs/astropy/astropy/units/core.pyi

```diff
@@ -0,0 +1,205 @@
+from _typeshed import Incomplete
+from typing import Any
+from typing_extensions import TypeAlias
+
+from astropy.units.equivalencies import Equivalency
+from astropy.utils.exceptions import AstropyWarning
+
+_Equivalences: TypeAlias = list[Equivalency]
```

Comment:
This is used in a few argument types, where it is probably too narrow a type. For example, `set_enabled_equivalencies` seems to accept any iterable.